### PR TITLE
Lua de-dotting filter

### PR DESCRIPTION
**Pull request by Teneroy**
**Description**

Changes proposed in this pull request:

- Moved lua script to separate file 
- Added filter  
- Added tests

**Related issue(s)**

https://github.com/kyma-project/kyma/issues/14310

### CODE REVIEW

**Review by chrkl — CHANGES_REQUESTED** [review]

File: components/telemetry-operator/internal/fluentbit/config_builder.go [hunk 1]
```diff
@@ -36,6 +36,11 @@ Emitter_Mem_Buf_Limit %s`
 name                  record_modifier
 match                 %s.*
 Record                cluster_identifier ${KUBERNETES_SERVICE_HOST}`
+	LuaDeDotFilterTemplate string = `
+name                  lua
+match                 %s.*
+script 				  /fluent-bit/scripts/filter-script.lua
```
[the remark this answers is not in the document]
Comment (dennis-ge) [reply components/telemetry-operator/internal/fluentbit/config_builder.go]: @chrkl I replaced the tabs with spaces. The intendation is on the same level as the `PermanentFilterTemplate` and `EmitterTemplate`, should we reduce it to 1 space?

**Review by chrkl — APPROVED** [review]

**Pull request merged**